### ISSUE TITLE
Update for rollup 2.x

### ISSUE THREAD

**Issue by lazka** [NotWoods/rollup-plugin-consts#1]
npm complains, though this plugins seems to work just fine as is with rollup 2.x :)

```
rollup-plugin-consts@1.0.1 requires a peer of rollup@>=1.15.0 <2 but none is installed. You must install peer dependencies yourself.
```

**Comment by NotWoods** [comment]
Thanks for the heads up! I published version 1.0.2 which fixes the peer dependency.

**Comment by lazka** [comment]
Thanks!

And thanks for the useful plugin!
